src/client/components/SearchFilterComponent/SearchFilter.styles.css

```diff
@@ -0,0 +1,304 @@
+    border-style: solid;
+    border-width: 0.1rem;
+    border-color: #C4C4C4;
+    }
+
+.container  h2 img{
+   position:relative;
+   bottom: 0rem;
+   width: 10px;
+   left:14rem;
+   }
+
+hr{
+   width: 22rem;
+   color: #c9bcbc;
+   }
+.container .input {
+   background-color: #ffffff;
+   width: 320px;
+   height: 103px;
+}
+.container .input input{
+  margin-top: 0.7rem;
+  margin-left: 1.5rem;
+}
+.container .input b{
+  font-family: roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  margin-top: 0.7rem;
+  margin-left: 0.2rem;
+  
+}
+
+/* Date posted */
+
+.container .posted{
+  background-color: #ffffff;
+  width: 320px;
+  height: 53px;
+  margin-top: 0rem;
+}
+.container  h3{
+  margin-top: 0.7rem;
+  margin-left: 1.5rem;
+  font-family: 'Times New Roman', Times, serif; 
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+    line-height: 21px;
+    color:rgb(0, 0, 0);
+    width: 340px;
+    height: 40px;
+    background-color: #ffffff;
+    padding-top: 0.5rem;
+    padding-left: 2.5rem;
+    position: relative;
+    right: 2.1rem;
+    border-style: solid;
+    border-width: 0.1rem;
+    border-color: #C4C4C4;
+    }
+    .container  h3 img{
+     position:relative;
+     bottom: 0rem;
+     width: 10px;
+     left:10rem;
+    }
+    .container .inputDate {
+     background-color: #ffffff;
+     width: 320px;
+     height: 123px;
+    }
+    .container .inputDate input{
+     margin-top: 0.7rem;
+     margin-left: 1.5rem;
+    }
+
+    .container .inputDate b{
+     font-family: roboto;
+     font-style: normal;
+     font-weight: normal;
+     font-size: 15px;
+     line-height: 21px;
+     margin-top: 0.7rem;
+     margin-left: 0.2rem;
+    }
+
+    /* minimum evaluation */
+
+    .container .minimum{
+     background-color: #ffffff;
+     width: 320px;
+     height: 53px;
+     margin-top: 0rem;  
+    }
+
+    .container  h4{
+     margin-top: 0.7rem;
+     margin-left: 1.5rem;
+        font-family: 'Times New Roman', Times, serif;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 20px;
+        line-height: 21px;
+        color:rgb(0, 0, 0);
+        width: 340px;
+        height: 40px;
+        background-color: #ffffff;
+        padding-top: 0.5rem;
+        padding-left: 2.5rem;
+        position: relative;
+        right: 2.1rem;
+        border-style: solid;
+        border-width: 0.1rem;
+        border-color: #C4C4C4;
+        }
+
+        .container  h4 img{
+            position:relative;
+            bottom: 0rem;
+            width: 10px;
+            left:5rem;
+        }
+
+        .container .inputMinimum {
+            background-color: #ffffff;
+            width: 320px;
+            height: 163px;
+        }
+       
+        .container .inputMinimum input{
+            margin-top: 0.7rem;
+            margin-left: 1.5rem;
+        }
+
+        .container .inputMinimum b{
+            font-family: roboto;
```

Comment:
You probably do not need it, but you can check
```suggestion

```

Reply:
i am implementing the same kind of letter as in the mockup. which should be the correct kind?

![css](https://user-images.githubusercontent.com/66638147/107790960-ad2c3200-6d53-11eb-86db-9a29923241e3.PNG)